### ISSUE TITLE
Error whilst Building (All dependencies met)

### ISSUE THREAD

**Issue by ParadauxDev** [spastorelli/dojohub#1]
Dependencies installed in c:\Go\src\github.com
dgrijalva/jwt-go
gorilla/websocket
golang/glog
spastorelli/app
Fork installed in C:\Users\rerri\workspace\dojohub

C:\Users\rerri\workspace\dojohub\app>**go build dojohub.go**
**msg_hub.go:49: invalid operation: token.Claims["aud"](type jwt.Claims does not support indexing)**
**msg_hub.go:59: not enough arguments to return**
**msg_hub.go:82: invalid operation: parsedToken.Claims["sub"](type jwt.Claims does not support indexing)**
**msg_hub.go:83: invalid operation: parsedToken.Claims["aud"](type jwt.Claims does not support indexing)**

**Comment by spastorelli** [comment]
Thanks for reporting this.

At the time I added the JWT tokens validation to `msg_hub.go` (around May 2016), `dgrijalva/jwt-go` latest revision was [`40bd0f3b4891a9d7f121bfb7b8e8b0525625e262`](https://github.com/dgrijalva/jwt-go/tree/40bd0f3b4891a9d7f121bfb7b8e8b0525625e262).

A good few changes to `dgrijalva/jwt-go` have been made since; however I haven't made anymore changes to `msg_hub.go`.

I have just pushed https://github.com/spastorelli/dojohub/commit/88d746f245db4311d2e25800812a38695d3fcecf to set the fixed dependency of `dgrijalva/jwt-go` in the meantime I update `msg_hub.go` to use the latest revision of `jwt-go` (**note:** also added some instructions to build and run the DojoHub).

I will create a separate issue to address the needed changes to `msg_hub.go`.
